src/ed25519.rs

```diff
@@ -549,7 +551,7 @@ impl ExpandedSecretKey {
     /// # fn main() { }
     /// ```
     pub fn from_secret_key<D>(secret_key: &SecretKey) -> ExpandedSecretKey
-            where D: Digest<OutputSize = U64> + Default {
+            where D: Digest<OutputSize = U64> + FixedOutput + Default {
```

Comment:
I don't think you need to add `FixedOutput` here, it's already part of the `Digest` trait.

Reply:
```suggestion
            where D: Digest<OutputSize = U64> + Default {
```